android/src/main/java/com/reactnativestripesdk/utils/Mappers.kt

```diff
@@ -346,6 +346,7 @@ internal fun mapFromPaymentMethod(paymentMethod: PaymentMethod): WritableMap {
     it.putString("fingerprint", paymentMethod.card?.fingerprint)
     it.putString("preferredNetwork", paymentMethod.card?.networks?.preferred)
     it.putArray("availableNetworks", paymentMethod.card?.networks?.available?.toList() as? ReadableArray)
+    it.putBoolean("isThreeDSecureSupported", paymentMethod.card?.threeDSecureUsage?.isSupported ?: false)
```

Comment:
let's match the native libraries and make this a dictionary, like this:
```
it.putMap("ThreeDSecureUsage", WritableNativeMap().also { threeDSecureUsageMap ->
  it.putBoolean("isSupported", paymentMethod.card?.threeDSecureUsage?.isSupported ?: false)
})
```

Reply:
Agree with that even it looks like the native type will never be update, it's coherent to be in sync